compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -102,6 +101,7 @@ pub(crate) fn eval_nullary_intrinsic<'tcx>(
             | ty::Never
             | ty::Tuple(_)
             | ty::Error(_) => ConstValue::from_machine_usize(0u64, &tcx),
+            ty::TyAlias(..) => bug!("unexpected TyAlias in eval_nullary_intrinsic"),
```

Comment:
Why are there interpreter changes in a PR that supposedly only does something to the rustdoc HTML  output...?

Reply:
O.o

I pushed on the wrong branch...